src/Compilers/CSharp/Portable/Compilation/SyntaxAndDeclarationManager.cs

```diff
@@ -530,7 +562,7 @@ public SyntaxAndDeclarationManager ReplaceSyntaxTree(SyntaxTree oldTree, SyntaxT
             }
```

Comment:
>} [](http://example.com/codeflow?start=12&length=1)

It looks like we are not taking advantage of ```lastComputedMemberNames``` on this branch. Is this intentional?

Reply:
Yes.  `#load` directives are not a common scenario, and preserving member names in these case for edits to those directives is not something there is any data to support investing in.